### ISSUE TITLE
Matlab error: Unexpected Standard exception from MEX file (MacOS)

### ISSUE THREAD

**Issue by annanas5** [pyomeca/ezc3d#235]
Hi! I am trying to install on macos 12 to be able to open .c3d in matlab. I have compiled using cmake and added the path to matlab, so that it can find the function. I have installed the code in /Users/annakoek/Documents/MATLAB. 

However, I get a matlab error saying:

Unexpected Standard exception from MEX file.
What() is:Could not open the c3d file: unspecified iostream_category error

Could you maybe explain to me what this means and how I can fix this?

Thanks!!

**Comment by annanas5** [comment]
Nevermind, I've worked it out!

**Comment by pariterre** [comment]
Hi @annanas5 
Could you explain what was the problem and how you fixed it? If anyone has the same issue, they will know how to solve it :)

**Comment by annanas5** [comment]
I think something went wrong with my .c3d file when downloading. I downloaded it again and the issue was fixed. Could have been an internet stability problem causing problems with the download, because the first download was on a very unstable connection. I was more or less looking in the wrong place to solve the problem. My fellow student uses the same .c3d file so I never thought to look at whether that file could have caused the problem, but in the end it did.
